### PR TITLE
Tickets/dm 31557

### PR DESCRIPTION
**Pull request by jbkalmbach**
Make changes needed to work with `ts_wep` v2.0.

### CODE REVIEW

File: doc/uml/utilsClass.uml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@startuml
+CreatePhosimDonutTemplates ..> MapSensorNameAndID
+@enduml
```
Comment (teweitsai) [inline doc/uml/utilsClass.uml]: Please add a new line after this. Thanks!

File: doc/uml/utilsClass.uml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+@startuml
+CreatePhosimDonutTemplates ..> MapSensorNameAndID
```
Comment (teweitsai) [inline doc/uml/utilsClass.uml:2]: I think you have another class in this module: `SensorWavefrontError`.

File: python/lsst/ts/phosim/OpdMetrology.py [hunk 3]
```diff
@@ -24,26 +24,45 @@
...
 class OpdMetrology(object):
     def __init__(self):
         """Initialization of OPD metrology class.
 
         OPD: Optical path difference.
+
+        Parameters
+        ----------
+        instName : `str`
+            Instrument name. Valid options are 'comcam or 'lsstfam'.
```
Comment (teweitsai) [inline python/lsst/ts/phosim/OpdMetrology.py]: I do not think you have this `instName` in `__init__()`. Do you?

**Review by teweitsai — APPROVED** [review]
This PR looks good to me. I saw you also import the following modules:
1. lsst.geom
2. lsst.afw
3. lsst.obs.base

Please update the `ups/ts_phosim.table` to reflect this before the merge.

Thanks!

**Comment by jbkalmbach** [comment]
Going to merge now that I have verified that closed loop runs correctly for comcam and lsstfam with changes in this PR.

**Pull request merged**